go.mod

```diff
@@ -18,5 +18,7 @@ require (
 	github.com/sparrc/go-ping v0.0.0-20190613174326-4e5b6552494c
 	github.com/stretchr/testify v1.5.1
 	github.com/vishvananda/netlink v1.1.0
+	golang.org/x/crypto v0.0.0-20190617133340-57b3e21c3d56 // indirect
```

Comment:
are these and the sum changes part of this change?

Reply:
Yea, buildkite was complaining

Reply:
This occurs when that auto version bump tool creates PRs. It doesn't do a tidy.